src-plugins/manualRegistration/manualRegistration.h

```diff
@@ -19,6 +19,11 @@
 
 class manualRegistrationPrivate;
 
+namespace TransformName
+{
+    enum Value {RIGID,AFFINE};
+}
```

Comment:
Seems like you are not using this enum in this class. You are using it in the toolbox (the values RIGID and AFFINE I mean).

I would have used it more in this class, like in the *switch* of the ```update()``` function and not necessarily as index of the combobox in the toolbox.